tests/x11/gnomecase/login_test.pm

```diff
@@ -31,7 +32,7 @@ sub run {
     $self->auto_login_alter;
     my $ov = get_var('NOAUTOLOGIN');
     set_var('NOAUTOLOGIN', '');
-    power_action('reboot');
+    power_action('reboot', keepconsole => 1) if is_opensuse;
```

Comment:
Hmm, looks like you wanted to set keepconsole if it's opensuse, but I guess we need to reboot not only for openSUSE, as we wait for boot in the next line.

Reply:
Yes, they all both reboot. I re-commit it. But it should keep console if using power_action to reboot on Tumbleweed.